### ISSUE TITLE
[offtopic] iGPU crash can be related to the design from device manufacturer

### ISSUE THREAD

**Issue by kurikomoe** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#48]
TLDR;
I suggest that the bug reports should carry the device brand name, such as GPD win3 or GX1PRO, etc.

-------

In GPD Win3 and OneXPlayer Chinese QQ Group, the crash reports seem to be highly corresponding to the QC of the manufacturing. 

For GPD win3, 1135g7 version seems to meet less crash. When things come to 1165g7, it varies a lot. Some report that the crash makes it hardly playable, while some can play smoothly.

For OneXPlayer, a person related to the manufacturer has shown that Dark Souls 3 can be played without crash at least 3 hours with ac supply.

I doubt that the power supply can be part of the cause of these crashes which based on an analysis in Chinese Baidu Tieba:  [关于TDP和GPU性能的初步测试(a basic test on TDP and GPU performance)](https://tieba.baidu.com/p/7314745797). 
Maybe when the CPU eats more energy, the iGPU cannot get enough to work properly, especially when the power supply is not stable.

Also, it is reported that turn off turbo frequency and hyperthreading can reduce the crash.

So, I highly recommend that the Bug report should carry with the device brands, such as GPD win3 or balabala brand, etc.  In this way, we can distinguish the iGPU driver bug from the improper design of the motherboard.

**Comment by Valkirie** [comment]
I believe that's a nice idea. I've seen issues submitted here that looked a lot like a faulty VR (Voltage Regulator).
I'll update the issue template accordingly.

**Comment by IGCIT** [comment]
eh!
i thought about this too, i just forgot this and i had not much time

an update to the IGCIT Helper is planned to include more info and i would like to push more its usage so that we have more precise data instead of "latest" etc.

yeah, these devices have some hw faults

thank you :)

**Comment by Valkirie** [comment]
> yeah, these devices have some hw faults
GPD recommending to lower GPU frequency to 800Mhz on some products is the biggest RED flag I've seen in a long time.

**Comment by kurikomoe** [comment]
I have pre-ordered OneXPlayer in China and will get one by May 12th (UTC+8). Hopefully, I can recheck some of the games listed in issues on the OneXPlayer 1165g7 version to verify the crash problems.

(Though the problem is that all my games are consoles version, I indeed have few games on Steam, LOL)

**Comment by Valkirie** [comment]
> I have pre-ordered OneXPlayer in China and will get one by May 12th (UTC+8). Hopefully, I can recheck some of the games listed in issues on the OneXPlayer 1165g7 version to verify the crash problems.
> 
> (Though the problem is that all my games are consoles version, I indeed have few games on Steam, LOL)

Looking forward to hear from you then. The more devices we have, the more we can differentiate false positive (from faulty device, bad luck, etc...) from true issue.

**Comment by IGCIT** [comment]
closing this now so we keep focus on game issues, thank you :)
